src/pages/iou/IOUModal.js

```diff
@@ -0,0 +1,173 @@
+import React, {Component} from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import ROUTES from '../../ROUTES';
+import {redirectToLastReport} from '../../libs/actions/App';
+import IOUAmountPage from './steps/IOUAmountPage';
+import IOUParticipantsPage from './steps/IOUParticipantsPage';
+import IOUConfirmPage from './steps/IOUConfirmPage';
+import Header from '../../components/Header';
+import styles from '../../styles/styles';
+import Icon from '../../components/Icon';
+import getPreferredCurrency from '../../libs/actions/IOU';
+import {Close, BackArrow} from '../../components/Icon/Expensicons';
+
+/**
+ * IOU modal for requesting money and splitting bills.
+ */
+const propTypes = {
+
+    /* Onyx Props */
+    // Url currently in view
+    currentURL: PropTypes.string,
+
+    // IOU modal data
+    iouData: PropTypes.shape({
+        isLoadingCurrency: PropTypes.bool.isRequired,
+        isLoadingParticipants: PropTypes.bool.isRequired,
+        createReportInProgress: PropTypes.bool.isRequired,
```

Comment:
Each of these nested props needs a comment

Reply:
Resolving as this no longer applies